### PR TITLE
style(video): fixing ondemand video page margins

### PR DESCRIPTION
**Pull request by runrog**
Adding some buffer room around embedded videos and on on-demand lp's

### CODE REVIEW

**Comment by runrog** [comment]
[run ci]

File: styleguide/derek/pattern-components/video/_video-embed.scss [hunk 1]
```diff
@@ -9,14 +9,10 @@
 }
 
 .videoEmbed-wrapper {
-  margin-bottom: 10px;
+  margin-bottom: 40px;
```
Comment (xinbin) [inline styleguide/derek/pattern-components/video/_video-embed.scss]: 🤠 perfect!

File: styleguide/derek/pattern-components/video/_video-embed.scss [hunk 2]
```diff
@@ -9,14 +9,10 @@
...
   position: relative;
 }
 
-.embed-responsive {
```
Comment (xinbin) [inline styleguide/derek/pattern-components/video/_video-embed.scss]: take it this was not being used anywhere? I'm just leaving comments to prove I looked at these things okay!

**Comment by xinbin** [comment]
+1+1

**Comment by runrog** [comment]
[run ci]

**Pull request merged**